index.html

```diff
@@ -0,0 +1,172 @@
+                <li><img src="./images/conversation.svg" alt="Conversation"></li>
+                <li class="headingList"><h1>Forum</h1></li>
+                <li><p> Have the time to share your thoughts
+                    and opinions with experts for
+                    each topic</p></li>
+            </ul>
+            <ul class="firstDetail">
+                
+                <li><img src="./images/workshop.svg" alt="Workshop"></li>
+                <li><h1>Workshop</h1></li>
+                <li><p> It's about shifting the mindset,medium and small scale farming can generate income.</p></li>
+
+            </ul>
+            <ul class="firstDetail">
+                
+                <li><img src="./images/Group.png" alt="Workshop"></li>
+                <li><h1>Equality</h1></li>
+                <li><p> Reducing cultural and gender norms as barriers to equality; changing laws to improve equality</p></li>
+
+            </ul>
+
+        </div>
+        <ul class="buttonList">
+            <li><button type="button">Join Women Summit 2021</button></li>
+        </ul>
+        
+        <a href="#">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <section class="featuredSpeaker">
+        <h1>Featured Speaker</h1>
+        <hr>
+        <div class="speakerDetail" id="dynamicList">
+        </div>
+        <ul class="buttonList">
+            <li><button class="more-btn" id="more-btn" type="button">More</button></li>
+        </ul>
+        
+    </section>
+    <section class="partner">
+        <h1>
+            Partner
+        </h1>
+        <hr>
+        <ul class="firstPartner">
+            <li><img src="./images/third.svg" alt="mozilla"></li>
+            <li><img src="./images/secondli.svg" alt="google"></li>
+            <li><img src="./images/firstli.svg" alt="Naver"> </li>
+        </ul>
+        <ul class="secondPartner">
+            <li><img src="./images/fourth.svg" alt="fourth"></li>
+            <li><img src="./images/fifth.png" alt="fifth"></li>
+        </ul>
+
+        <ul class="desktopPartner">
+            <li><img src="./images/third.svg" alt="mozilla"></li>
+            <li><img src="./images/secondli.svg" alt="google"></li>
+            <li><img src="./images/firstli.svg" alt="Naver"> </li>
+            <li><img src="./images/fourth.svg" alt="fourth"></li>
+            <li><img src="./images/fifth.png" alt="fifth"></li>
+        </ul>
+    </section>
+    <footer>
+       <div class="footerMainul">
+           
+           <ul class="footerFirstul">
+               <li>creative</li>
+               <li>commons America</li>
+           </ul>
+           <ul class="footerSecondul">
+               <li>2021 Creative Commons America</li>
+               <li>Some Rights Reserved</li>
+           </ul>
+           <ul class="footerthirdul">
+            <li>2021 Creative Commons America Some Rights Reserved</li>
+            <li>妇女能顶半边天妇女能顶半边天妇女能顶半边天妇女能顶半边天</li>
+        </ul>
+    </div>
+    </footer>
```

Comment:
- [ ] KIndly use the correct colour for the lower footer. It shouldn't be white but `#d3d3d3`